lib/ui/baseUI.js

```diff
@@ -43,12 +51,12 @@ UI.prototype.onForceClose = function () {
 UI.prototype.close = function () {
   // Remove events listeners
   this.rl.removeListener('SIGINT', this.onForceClose);
-  process.stdin.removeListener('keypress', this.onKeypress);
+  this._input.removeListener('keypress', this.onKeypress);
   process.removeListener('exit', this.onForceClose);
 
   // Restore prompt functionnalities
   this.rl.output.unmute();
-  process.stdout.write('\x1B[?25h'); // show cursor
+  this._output.write('\x1B[?25h'); // show cursor
```

Comment:
FWIW you can delete this line. I forgot to remove it when I moved cursor support to cli-cursor.